scio-bigtable/src/main/scala/com/spotify/scio/bigtable/Bigtable.scala

```diff
@@ -17,8 +17,14 @@
 
 package com.spotify.scio.bigtable
 
-import com.google.cloud.bigtable.dataflow.CloudBigtableOptions
+import java.util.concurrent.TimeUnit
+
+import com.google.bigtable.repackaged.com.google.com.google.bigtable.admin.v2.{BigtableInstanceAdminGrpc, Cluster, ListClustersRequest}
+import com.google.cloud.bigtable.dataflow.{CloudBigtableConfiguration, CloudBigtableOptions}
 import com.spotify.scio.ScioContext
+import org.joda.time.Duration
+
+import scala.collection.JavaConverters._
 
 /** Utilities for Bigtable. */
 object Bigtable {
```

Comment:
Yeah sure. I hope the Bigtable team will incorporate this themselves at some point as I feel its pretty important.